src/c++/vdo/base/slab-depot.c

```diff
@@ -2288,14 +2297,21 @@ static void load_reference_block(struct vdo_waiter *waiter, void *context)
 static void load_reference_blocks(struct vdo_slab *slab)
 {
 	block_count_t i;
+	u64 step = slab->allocator->refcount_blocks_per_big_vio;
+	struct vio_pool *pool = slab->allocator->refcount_big_vio_pool;
+
+	if (!pool) {
+		pool = slab->allocator->vio_pool;
+		step = 1;
+	}
 
 	slab->free_blocks = slab->block_count;
-	slab->active_count = slab->reference_block_count;
-	for (i = 0; i < slab->reference_block_count; i++) {
+	slab->active_count = DIV_ROUND_UP(slab->reference_block_count, step);
```

Comment:
I kind of feel like it's simpler here (less math) to not change the meaning of active_count, and simply say a large bio accounts for several active reference counts blocks. The vio inherently knows how many blocks it covers anyway.